S21/osamaibraheem11/s21.py

```diff
@@ -0,0 +1,26 @@
+import csv
+
+vowels = ["A","E","I","O","U","Á","É","Í","Ó","Ú"]
+
+def LowerCase(word,language): 
+  if ('ga' in language) and (word.startswith('n') or word.startswith('t')) and word[1:2] in vowels:
+    return word[:1].lower()+"-"+word[1:].lower()
+  if 'en' in language:
+    return word.lower()
+  if ('tr' or "az" in language):
```

Comment:
Should separate into two boolean expressions ('tr' in language or 'az' in language).   (Noting of course that the "in" needs to be fixed as well per the comment above).